core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -292,9 +293,6 @@ public int getStatus(
             int lo,
             int hi
     ) {
-        if (writerPool.exists(tableName)) {
-            return TableUtils.TABLE_EXISTS;
```

Comment:
this branch was never taken

Reply:
👍 that's the fix, right ?

Reply:
not exactly. removal of these lines does not change the behaviour as the branch was never taken. it was basically just dead-code. 

this actual fix is this: https://github.com/questdb/questdb/pull/2046/files#diff-f7e6db2048ee1913e5e8f5f3973ad0fc355fe9c690be667c4f65f9af05162194R137 - instead of relying on writer pool we check the table status.